### DISCUSSION TITLE
PSD to HTML projects

### DISCUSSION THREAD

**Discussion by ajaxStardust** [tannerhelland/PhotoDemon#462] [Q&A] [answered]
Hi there. I recently discovered **_PhotoDemon_**. As someone who's used graphic design software since the 1990's (oh gawd, i'm old!), I find it quite awesome-- actually authored a [brief blog post](https://wordpresscenter.net/2022/09/21/web-development-essential-tools-software/) about it for you! 
:)

I wonder if PhotoDemon offers any extensions / options to get started on a PSD to HTML conversion project. Some apps that open the PSD files will show the different sections (e.g. as would be perhaps HTML submenus, headings H1, H2, whatever) I presume you are familiar w/ what all that is, but just in case, i suppose [this is an informative article](https://exportkit.com/plugin/environments/html5/convert-a-psd-to-html5) about it maybe for a software developer such as yourself? or [this](https://xirosoft.com/convert-psd-html/) for a half-ash web dev, such as myself (which I might follow later today?). 

Edit: found the link i was looking for - There's free [online tools](https://psd2htmlconverter.com) that do it, but I wasn't super happy with the results I got.

Just thought I'd run it by PhotoDemon to see if the developer, or perhaps an enthusiastic plugin developer might have some groovy suggestions! Maybe it's already an option in PhotoDemon, but I haven't found it. :)

Best regards!
~JS

**Comment by abhishekZom** [comment, marked as answer]
PhotoDemon does not have the capability to convert PSD (Photoshop Document) files to HTML (HyperText Markup Language). However, there are other tools and software available that can help with the conversion of PSD to HTML.

One option is to use a dedicated PSD to HTML conversion tool. These tools often have the ability to import PSD files and automatically generate the corresponding HTML and CSS code. Some popular options for converting PSD to HTML include Adobe Dreamweaver, PSD2HTML, and Sketch2HTML.

Another option is to use a design tool that supports HTML export. Some design tools, such as Adobe XD and Figma, allow you to design your layouts and then export them to HTML and CSS.

Hope this helps!
